client/src/app/app.routes.ts

```diff
@@ -25,8 +27,8 @@ export const ROUTES: Routes = [
   {path: 'home', component: WelcomeComponent},
   {path: 'registration', component: RegistrationComponent, canActivate: [UserIsLoggedInGuard]},
   {path: 'projects', component: MyProjectsComponent, canActivate: [UserIsLoggedInGuard]},
-  {path: 'projects/register', component: ProjectRegistrationFormComponent, canActivate: [UserIsLoggedInGuard]},
-
+  {path: 'projects/register', component: AutofillProjectFormComponent, canActivate: [UserIsLoggedInGuard]},
+  {path: 'projects/new', component: ProjectRegistrationFormComponent, canActivate: [UserIsLoggedInGuard]},
```

Comment:
Why a new route for this behavior? Why can't the autofill be part of ProjectRegistrationFormComponent?

Reply:
okay so I think it was because I couldn't figure out a good way to pass the input doi to `ProjectRegistrationFormComponent`, but in retrospect we could just pass it via events, without having a new route

But also, having two different routes meant that the user could press the back button on the browser to get back to the input your doi screen, from the project form page, instead of pressing back and reaching the home page, which is a flow I'd expect as a user

what do you think?

Either way, let's add this as an improvement in the next iteration. There's some functional feedback I need to incorporate as well.

Reply:
Yeah that makes sense. Fine by me